python/lsst/pipe/supertask/config.py

```diff
@@ -25,13 +25,15 @@
 
 from __future__ import absolute_import, division, print_function
 
-__all__ = ["QuantumConfig", "InputDatasetConfig", "OutputDatasetConfig",
-           "InitInputDatasetConfig", "InitOutputDatasetConfig",
-           "SuperTaskConfig"]
+__all__ = ["InputDatasetField", "OutputDatasetField", "InitInputDatasetField",
+           "InitOutputDatasetField", "QuantumConfig", "InputDatasetConfig",
+           "OutputDatasetConfig", "InitInputDatasetConfig",
+           "InitOutputDatasetConfig", "SuperTaskConfig"]
 
 # -------------------------------
```

Comment:
This is not your code but we don't generally start our python code with these kinds of comments. @andy-slac I see that supertask is doing this everywhere though.

Reply:
@timj Yes, you should blame me for this, it comes from a common template that I used in many projects. Is this a bad style?

Reply:
It might not be bad, it's just different to every other python package we work on.

Reply:
Not too different from qserv, though :smile: